prod-aws/finance/direct_debits.tf

```diff
@@ -14,6 +14,7 @@ resource "kafka_topic" "dd_run_reconciliation_events" {
   replication_factor = 3
   partitions         = 10
   config = {
+    "max.message.bytes" = "1048576"
```

Comment:
As precaution, can we do it in `prod` later on?

Reply:
ok